Lib/test/test_fcntl.py

```diff
@@ -12,7 +12,6 @@
 fcntl = import_module('fcntl')
 
 
-# TODO - Write tests for flock() and lockf().
```

Comment:
Is the part about flock() still relevant?

Reply:
https://github.com/python/cpython/blob/master/Lib/test/test_fcntl.py#L127

test_flock already exists.
But if the test is not enough, the comment should be remain